### PR TITLE
dynamics_for_different_voices

### PR DESCRIPTION
**Pull request by RomanPudashkin**
Resolves: https://github.com/musescore/MuseScore/issues/19137
Resolves (for MS Basic): https://github.com/musescore/MuseScore/issues/19207

### CODE REVIEW

**Review by Eism — APPROVED** [review]

File: src/engraving/dom/dynamic.cpp [hunk 1]
```diff
@@ -643,6 +648,8 @@ PropertyValue Dynamic::propertyDefault(Pid id) const
         return DynamicSpeed::NORMAL;
     case Pid::PLAY:
         return true;
+    case Pid::APPLY_TO_ALL_VOICES:
+        return false;
```
Comment (Jojo-Schmitz) [inline src/engraving/dom/dynamic.cpp:652]: Shouldn't the default rather be `true`, just like it was before your PR?

File: src/engraving/dom/hairpin.cpp [hunk 2]
```diff
@@ -537,6 +542,9 @@ PropertyValue Hairpin::propertyDefault(Pid id) const
     case Pid::PLAY:
         return true;
 
+    case Pid::APPLY_TO_ALL_VOICES:
+        return false;
```
Comment (Jojo-Schmitz) [inline src/engraving/dom/hairpin.cpp:546]: Shouldn't the default rather be `true`, just like it was before your PR?

File: src/engraving/dom/dynamic.h [hunk 3]
```diff
@@ -125,6 +128,7 @@ class Dynamic final : public TextBase
     DynamicType m_dynamicType = DynamicType::OTHER;
     Expression* m_snappedExpression = nullptr;
     bool m_playDynamic = true;
+    bool m_applyToAllVoices = false;
```
Comment (Jojo-Schmitz) [inline src/engraving/dom/dynamic.h:131]: Shouldn't the default rather be `true`, just like it was before your PR?

File: src/engraving/dom/hairpin.h [hunk 4]
```diff
@@ -106,6 +106,7 @@ class Hairpin final : public TextLineBase
     bool _singleNoteDynamics;
     ChangeMethod _veloChangeMethod;
     bool _playHairpin;
+    bool m_applyToAllVoices = false;
```
Comment (Jojo-Schmitz) [inline src/engraving/dom/hairpin.h:109]: Shouldn't the default rather be `true`, just like it was before your PR?

File: src/engraving/dom/dynamic.cpp (hunk 1 above)
Comment (RomanPudashkin) [reply src/engraving/dom/dynamic.cpp:652]: It's false by design. If you don't like this new behavior, you can discuss it with @avvvvve in this issue
https://github.com/musescore/MuseScore/issues/19137

**Comment by RomanPudashkin** [comment]
I'm going to close it and open a few smaller PRs

**Pull request closed without merging**